docs/dev/config.md

```diff
@@ -1,14 +1,15 @@
-# Configurationmanagement
+# Configuration and Featuers 
 
-All Components are implemented in the package `org.eclipse.winery.common.configuration`.
-The configuration is YAML based and called "winery.yml". The default Configuration is contained in
-the resource folder.
+Implemented in the package `org.eclipse.winery.common.configuration`.
+The configuration is YAML based and called `winery.yml`.
+The default Configuration is contained in the resource folder.
 
 ## Adding new Features
-To add a new feature to the configuration one has to simply add it hierarchically under the features tab in the winery.yml file.
+
+To add a new feature to the configuration one has to simply add it hierarchically under the features tab in the `winery.yml` file.
 This has to be done both in the winery.yml file in the filesystem and in the default configuration winery.yml file in the resources folder.
 
-```yaml
```

Comment:
Why did you remove the language specification? If you add it, it enables code highlighting:

Whithout ` ```yaml`
```
ui:
  features:
    splitting: true
    autocommit: false
    username: default
```

With:
```yaml
ui:
  features:
    splitting: true
    autocommit: false
    username: default
```